internal/pkg/aws/ec2/ec2.go

```diff
@@ -103,16 +130,21 @@ func (c *EC2) SecurityGroups(filters ...Filter) ([]string, error) {
 
 func (c *EC2) subnets(filters ...Filter) ([]*ec2.Subnet, error) {
 	inputFilters := toEC2Filter(filters)
+	var subnets []*ec2.Subnet
+	nextToken := aws.String("")
 
-	response, err := c.client.DescribeSubnets(&ec2.DescribeSubnetsInput{
-		Filters: inputFilters,
-	})
-
-	if err != nil {
-		return nil, fmt.Errorf("describe subnets: %w", err)
+	for nextToken != nil {
+		response, err := c.client.DescribeSubnets(&ec2.DescribeSubnetsInput{
```

Comment:
Don't you need to pass in the `nextToken`?

Reply:
damn i forgot